src/bumerang/handler/profileedithandler.py

```diff
@@ -0,0 +1,33 @@
+from bumerang.handler.bumerangrequesthandler import BumerangRequestHandler
+from bumerang.error import BumerangError
+
+
+class ProfileEditHandler(BumerangRequestHandler):
+    """ Allows the user to edit their profile after it's created """
+
+    def post(self, id):
```

Comment:
The implementation of this looks good. In terms of UX, it may be nice to return a `400 bad request` if the user doesn't supply any of the attributes to update an item. We could also save a call to the database. The main purpose of this, is say the client sends in `deviceId` instead of `device_id`, the server returns a 200 and says the profile has been updated, because there were no errors or bugs in the code, but the client thinks the profile is updated, while the server does not. This doesn't really need to be done, at least not right now.